Gemfile

```diff
@@ -98,15 +98,15 @@ gem "premailer-rails", "~> 1.12.0", require: false
 gem "public_suffix", "~> 5.0.1"
 
 # Puma as app server
-gem "puma", "~> 6.0.2"
+gem "puma", "~> 6.3.0"
 
 # Make cracking a little bit harder
-gem "rack-attack", "~> 6.6.1"
+gem "rack-attack", github: "rocketreferrals/rack-attack", branch: "allow-rack-3"
```

Comment:
can't we use that branch in `rack/rack-attack`?

Reply:
done